pages/templates/partials/sign-up.html

```diff
@@ -2,7 +2,13 @@
     {% if not page.issuepage %}<div class="col-md-6 col-md-offset-3">{% endif %}
         <h3>
             {% if page.candidateendorsementpage %}
-                Join the fight to get {{ page.candidate.name }} elected.
+                Join the fight to get
+                {% if page.candidate %}
+                {{ page.candidate.name }}
+                {% else %}
+                {{ page.title }}
```

Comment:
If we're going to treat title as name needs to be clear to an admin when creating page. Can we add/override help-text on title field? Don't want `Join the fight to get Our Revolution Endorses Mike Connolly elected` by accident if admins misinterpret page title.

Reply:
@cjmabry it's a good idea but I'm not sure how to do this. See: https://github.com/wagtail/wagtail/issues/161

Reply:
Ok fine for now if we can't do it, let's remember to specify this in any documentation.

Reply:
Yeah I agree should be called out in our training/documentation.